src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -100,10 +100,12 @@ channels:
           - $ref: '#/components/messages/phoneState'
 
   lte_ota_message:
```

Comment:
It looks like you missed renaming the lte_ota_message to cellular_ota_message here.